pkg/runner/kubectl/describe_test.go

```diff
@@ -14,133 +14,158 @@ func TestDescribe(t *testing.T) {
 		collector *v1alpha1.Describe
 		want      *v1alpha1.Command
 		wantErr   bool
-	}{{
-		name:      "nil",
-		collector: nil,
-		want:      nil,
-		wantErr:   false,
-	}, {
-		name:      "empty",
-		collector: &v1alpha1.Describe{},
-		wantErr:   true,
-	}, {
-		name: "without resource",
-		collector: &v1alpha1.Describe{
-			ObjectLabelsSelector: v1alpha1.ObjectLabelsSelector{
-				Name: "foo",
+	}{
+		{
+			name:      "nil",
+			collector: nil,
+			want:      nil,
+			wantErr:   false,
+		},
```

Comment:
why did you change the indentation ?

Reply:
it changed automatically

Reply:
would be great ! the old indentation is more compact :)

Reply:
okay reverting back